OneCommunityGlobal/HighestGoodNetworkApp
# Anirudh SK finishing for Jay - PERMISSION: Modify Task X on Dashboard Permission

# Description
This PR enables the user with the View and Interact with Task “X” on Dashboards Permission to be able to access the X button on dashboard under task name and update the user.

## Related PRS (if any):
This frontend PR is related to the [#707](https://github.com/OneCommunityGlobal/HGNRest/pull/707) backend PR.

…

## Main changes explained:
- Added a new permission label for View and Interact with Task “X” on Dashboards.
- Added conditional logic so that the user with the permission can view the X button and update the user.

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log as owner user
5. go to Other Links -> Permissions Management -> Manage User Permissions -> Choose a user without View and Interact with Task "X" on Dashboards under Project Management, Work Breakdown Structures, Tasks -> click on Add  -> Scroll down then submit
6. verify that the user who was given the permission is now able to see the the X button on dashboard under each task.


## Screenshots or videos of changes:




https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/118868270/164a5615-5ef9-4de8-9abd-4c4179ca96b1




## Note:
**Unmute the video**
Please make sure you log out and log in again and then check in the account you are assigning the permission to
